client/my-sites/customer-home/main.jsx

```diff
@@ -96,14 +97,14 @@ const Home = ( {
 
 	const header = (
 		<div className="customer-home__heading">
+			<TrackResurrections />;
```

Comment:
Is this the place where we want to put this? Jetpack sites don't have a "My Home" page, so this just won't run for users whose primary (or only) site is a Jetpack site.

Reply:
Going to test against Jetpack and relocate appropriately.

Reply:
So I found the three other landing pages from the function getLoggedInLandingPage i.e
`/home/${ primarySiteSlug }`;
`/stats/${ primarySiteSlug }`;
and '/read';
And I would like to add the resurrection component to each of them.
However, I need guidance on the recommended way to include AsyncLoad to the files outside the client folder, e.g., in the HighlightCards component, as it's in the packages library outside the client library. This component is displayed on the landing page of jetpack sites
https://github.com/Automattic/wp-calypso/blob/2362c74ba7803cd595339cb16c80b2fbb6d06331/packages/components/src/highlight-cards/index.tsx

Reply:
I wouldn't recommend putting `<TrackResurrections />` inside a generic component that can be used anywhere.

Instead, I'd recommend putting it alongside the `<HighlightCards />` instances, even if we have to insert it in multiple places.